forest_plot_lib.py

```
@@ -438,18 +312,18 @@ def get_p_color(p):
         customdata = np.stack((
             self.data[self.ci_low_col], 
             self.data[self.ci_high_col], 
-            self.data['__display_p'] if has_pval else [None]*len(self.data), # Use formatted P for hover
+            self.data['__display_p'] if has_pval else [None]*len(self.data),
             self.data[self.ci_high_col] - self.data[self.ci_low_col]
         ), axis=-1)
 
         fig.add_trace(go.Scatter(
             x=self.data[self.estimate_col], y=y_pos,
             error_x=dict(type='data', symmetric=False, array=self.data[self.ci_high_col] - self.data[self.estimate_col], arrayminus=self.data[self.estimate_col] - self.data[self.ci_low_col], color='rgba(100,100,100,0.5)', thickness=2, width=4),
```

Ruff (0.14.10)

Unnecessary `dict()` call (rewrite as a literal)

Rewrite as a literal

(C408)